assets/css/query-builder.css

```diff
@@ -602,18 +614,105 @@ margin-bottom: 8em;
 }
 
 
-#search-history tbody button,
-#query-builder .actions button,
-#search-manager .history-controls button {
+.search-history table{
+  table-layout: fixed;
+}
+.search-history tbody button {
 	margin: 0 0.125em;
 	padding: 0;
-	background: transparent;
 }
 
 td.q {
 	cursor: pointer;
 }
 
+th.num,
+td.num{ width: 8em; }
+
+th.history-controls,
+td.history-controls{ width: 114px; }
+
+.search-history > h1,
+.search-history--actions{
+	display: inline-block;  
+}
+
+.search-history--actions{
+	padding-left: 0.4em;
+	position: relative;
+	top: 12px;
+}
+
+.search-history--actions::after{
+	content: "";
+	display: table;
+	clear: both;
+}
+
+.search-history .action{
```

Comment:
I think if we really want to do this -- and I'm only 50/50 on the value of it -- there are probably existing button groups in the pubmed styles we should leverage.